scripts/generate_schemas.py

```diff
@@ -277,9 +297,35 @@ def convertAvro(self, avdlFile):
             print("converting", avdlFile)
         if self.verbosity > 1:
             print("running:", " ".join(args))
-        # TODO silence the Java output unless we are in verbose mode by
-        # capturing stdout/stderr here
-        subprocess.check_call(args)
+        if self.verbosity > 1:
+            subprocess.check_call(args)
+        else:
+            devnull = open(os.devnull, 'w')
```

Comment:
It would be nice to use a `with` block here to ensure `devnull` is closed even if an error occurs.